### ISSUE TITLE
The information on the Segoe UI Emoji font family is inadequate

### ISSUE THREAD

**Issue by AlkisPis** [MicrosoftDocs/typography-issues#1080]
(Re: https://learn.microsoft.com/en-us/typography/font-list/segoe-ui-emoji)

1) There should be a list of **all the available emojis** in that font.
2) It should be indicated **how one can type them** in a Word document.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: a897815e-652e-d4ae-8f3d-979ee1ec0fcb
* Version Independent ID: 575e9a80-3e10-d850-eea7-d3c921779f82
* Content: [Segoe UI Emoji font family - Typography](https://learn.microsoft.com/en-us/typography/font-list/segoe-ui-emoji)
* Content Source: [typographydocs/font-list/Segoe-UI-Emoji.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/font-list/Segoe-UI-Emoji.md)
* Product: **typography**
* GitHub Login: @alib
* Microsoft Alias: **alib**

**Comment by fitojb** [comment]
I disagree on point 2. Why Word and not LibreOffice Writer, for instance? The Typography documentation is not Microsoft 365 documentation, so it is not its job to describe unrelated software's features or procedures.

**Comment by alib-ms** [comment]
Hi, 

The Segoe UI Emoji font is regularly updated in accordance with Unicode releases. When an Emoji is supported in Unicode, it is also supported in our fonts and products. In the event that an Emoji on our platform is missing, kindly use the Feedback Hub in Windows to report the issue.

Emojis can be incorporated into Office and other applications through various methods. However, the most convenient way to add emojis is through the emoji panel. To access the Windows emoji panel, simply click on a text input area in any Windows application and press either the Windows key and period (Windows+.) or the Windows key and semicolon (Windows+;). This action will prompt the emoji picker panel to appear. Once the panel is open, you can easily start adding emojis.
